### PR TITLE
Adding Brightspace/d2l-content-store to BSI.

### PR DESCRIPTION
**Pull request by qwangd2l**
Adding https://github.com/Brightspace/d2l-content-store to BSI.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -24920,6 +24958,22 @@
...
+    "redux": {
+      "version": "4.0.5",
+      "resolved": "https://registry.npmjs.org/redux/-/redux-4.0.5.tgz",
+      "integrity": "sha512-VSz1uMAH24DM6MF72vcojpYPtrTUu3ByVWfPL1nPfVRb5mZVTve5GnNCUV53QM/BZ66xfWrm0CTWoM+Xlz8V1w==",
+      "dev": true,
+      "requires": {
+        "loose-envify": "^1.4.0",
+        "symbol-observable": "^1.2.0"
+      }
+    },
```
Comment (dlockhart) [inline package-lock.json]: A little surprised to see redux in here... are you using it directly? I think we're hoping to align on [MobX for state management](https://blog.d2l.dev/2020/01/22/ui-statemanagement-mobx/) everywhere if we can.

Comment (qwangd2l) [reply package-lock.json]: Yeah we are using it directly but we're early in the project. We'll remove redux and add in a state management later.

File: package-lock.json [hunk 2]
```diff
@@ -4413,7 +4425,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#f607245c5d45eca546dcd08122db99c98f08305b",
+      "version": "github:Brightspace/d2l-rubric#0ae8c207a51dddedd4be068324f4379af3e128f0",
```
Comment (qwangd2l) [inline package-lock.json]: FYI @mpharoah-d2l

File: package-lock.json [hunk 3]
```diff
@@ -3666,10 +3672,11 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#d712e9eef25f343378e4d44eb1d30997dbd2b5cd",
+      "version": "github:BrightspaceHypermediaComponents/activities#a84cdec35aed54291281e032a1a0610e34495667",
```
Comment (qwangd2l) [inline package-lock.json]: FYI @mdgbayly @kieranderson

File: package-lock.json [hunk 4]
```diff
@@ -3652,7 +3658,7 @@
       }
     },
     "d2l-accordion": {
-      "version": "github:BrightspaceUI/accordion#e340d9746a87cab5b3e4673677f20dd15c6f664c",
+      "version": "github:BrightspaceUI/accordion#bc023ff9deddc34747a9f774577f841b1c3dd004",
```
Comment (qwangd2l) [inline package-lock.json]: FYI @pureooze

File: package-lock.json (hunk 2 above)
Comment (mpharoah-d2l) [reply package-lock.json]: This looks like its going backwards for some reason

Comment (qwangd2l) [reply package-lock.json]: Seems right to me?
https://github.com/Brightspace/d2l-rubric/releases
`0ae8c207a51dddedd4be068324f4379af3e128f0`

Comment (mpharoah-d2l) [reply package-lock.json]: I didn't make the release until later, so you might need to re-run `npm i`
Or just remove this change manually

Comment (qwangd2l) [reply package-lock.json]: Oh ok, sure I can remove this

File: package-lock.json (hunk 1 above)
Comment (cpacey) [reply package-lock.json]: We're definitely trying to align on MobX.  I'd like to understand the reasoning before you decide to use something different like Redux.

File: package-lock.json (hunk 2 above)
Comment (mpharoah-d2l) [reply package-lock.json]: Oh, I misread this. I think GitHub is just showing a bad diff or something? Since it should already be on 0ae...

Comment (mpharoah-d2l) [reply package-lock.json]: Yeah, it's just the GitHub machine running amok again. The "change" is changing it to what's already in master

Comment (qwangd2l) [reply package-lock.json]: > Yeah, it's just the GitHub machine running amok again. The "change" is changing it to what's already in master

Weird, I need to do another install anyway so I'll see if that fixes the diff.

**Pull request merged**